### ISSUE TITLE
AI_turn

### ISSUE THREAD

**Issue by R3mmurd** [luiscuriel-bit/Pong#1]
https://github.com/luiscuriel-bit/Pong/blob/ab7ccc61f395b2ed1cb493108f9765ab6f4d054c/pong.c#L94

I don't get the behavior of the variable `AI_turn`.

**Comment by R3mmurd** [comment]
By the way, using `if (true == true // evaluates to true)` or `if (false == true // evaluates false)` is redundant.

**Comment by luiscuriel-bit** [comment]
Good evening, AI_turn is a bool variable that serves to indicate two states: player 1's turn with true and player 2's turn with false.
For example, when AI_turn is true player 1's paddle is "turned on" by changing vy to align its y-coordinate with the ball, while paddle 2's speed is set to 0 so it remains stationary.

**Comment by luiscuriel-bit** [comment]
You are right, an unnecessary comparison is made. Excuse me, at the time it was easier for me to see it like this.

**Comment by luiscuriel-bit** [comment]
Also, AI_turn is set to true or false depending on the collision with player 1’s or player 2's paddle. This is done in pong.c at line 197 and line 213. I will add this to de CHANGELOG. I hope i made it more clear.
